handlers/metrics.go

```diff
@@ -52,6 +52,17 @@ func extractMetricsQueryParams(r *http.Request, q *prometheus.MetricsQuery) erro
 	if filters, ok := queryParams["filters[]"]; ok && len(filters) > 0 {
 		q.Filters = filters
 	}
+	if quantiles, ok := queryParams["quantiles[]"]; ok && len(quantiles) > 0 {
+		q.Quantiles = quantiles
```

Comment:
Minor one, if I put a wrong quantile here, a bad request (400) is sent ? or it falls into the default quantile value ?
(perhaps this is coded later, but I leave question at the handler)

Reply:
There's actually no validation at all neither here nor elsewhere, I'm going to add it.